docs/guides/integration-aws-entity-resolution.md

```diff
@@ -0,0 +1,197 @@
+
+| Step | Actor | Action |
+| --- | --- | --- |
+| 1 | Participant | Signs up for AWS. |
+| 2 | Participant | Signs UID2 contract and shares AWS Account ID with UID2 contact. |
+| 3 | Participant | Subscribes to UID2 on the Amazon Data Exchange. |
+| 4 | UID2 Administrator | Matches the AWS Account ID provided in Step 2 with the signup request received in Step 3 and approves the request. |
+| 5 | Participant | Stores input data in an Amazon S3 bucket. |
+| 6 | Participant | Sets up Entity Resolution and creates a matching workflow. |
+| 7 | Participant | Runs the matching workflow. |
+| 8 | AWS Entity Resolution | Reads input data from the S3 bucket. |
+| 9 | AWS Entity Resolution | Sends the input data to the Amazon Data Exchange using `POST /SendApiAsset`. |
+| 10 | Amazon Data Exchange | Sends the data to the API Gateway using `POST /uid2/assignment`. |
+| 11 | API Gateway | Sends the data to Lambda using `path=assignment`. |
+| 12 | Lambda for UID2 generation | Sends the data to the UID2 Public Operator for conversion to raw UID2 using `POST /identity/map`. |
+| 13 | UID2 Public Operator | Returns raw UID2s to Lambda for UID2 generation. |
+| 14 | Lambda for UID2 generation | Returns raw UID2s to the API Gateway. |
+| 15 | API Gateway | Returns raw UID2s to the Amazon Data Exchange. |
+| 16 | Amazon Data Exchange | Returns raw UID2s to AWS Entity Resolution. |
+| 17 | AWS Entity Resolution | Writes the output records to the S3 bucket including the raw UID2s. |
+
+## Initial Setup Steps
+
+The following are the initial steps for setting up your UID2 integration in AWS Entity Resolution:
+
+- [Create AWS Account](#create-aws-account)
+- [Create UID2 Account](#create-uid2-account)
+- [Subscribe to UID2 on AWS](#subscribe-to-uid2-on-aws)
+
+### Create AWS Account
+
+Sign up for an AWS account, or use an existing one.
+
+For details, see [Getting started: Are you a first-time AWS user?](https://docs.aws.amazon.com/accounts/latest/reference/welcome-first-time-user.html) in the AWS documentation.
+
+### Create UID2 Account
+
+Ask your UID2 contact to register you as a UID2 participant. If you're not sure who to ask, see [Contact Info](../getting-started/gs-account-setup.md#contact-info).
+
+Send your UID2 contact the AWS account ID that you'll use for your UID2 integration on Entity Resolution.
+
+### Subscribe to UID2 on AWS
+
+To participate in UID2 on the Amazon Data Exchange Marketplace, subscribe to the UID2 service. See [Unified ID 2.0 Operator on AWS](https://aws.amazon.com/advertising-marketing/solutions/uid2/).
+
+On receipt of your subscription request, the UID2 team do the following:
+- Review your subscription request.
+- Match your request with the AWS account ID that you sent in [Create UID2 Account](#create-uid2-account).
+- Approve the request.
+
+If you send your request via your UID2 contact, you'll receive a custom invitation to the UID2 private service listing on AWS, as shown in the following example.
```

Comment:
Its important to convey that this is a temporary arrangement  and the only way until we make the listing public on ADX. May be add more detail like this
> Currently, you need to send your request via your UID2 contact, then you'll receive a custom invitation to the UID2 private service listing on AWS, as shown in the following example, that you need to accept.

Reply:
Thx. Done, almost identical wording: "Currently, you need to send your request via your UID2 contact. Then, you'll receive a custom invitation to the UID2 private service listing on AWS, as shown in the following example, and must accept the invitation to continue."